ethereum/scripts/dumpRelayerConfig.js

```diff
@@ -3,38 +3,44 @@ const TOML = require('@iarna/toml');
 const fs = require('fs');
 const os = require('os');
 
-const Bridge = artifacts.require("Bridge")
-const ETHApp = artifacts.require("ETHApp")
-const ERC20App = artifacts.require("ERC20App")
-
-const dump = (bridge, ethApp, erc20App) => {
+const channelContracts = {
+    basic: {
+        inbound: artifacts.require("BasicInboundChannel"),
+        outbound: artifacts.require("BasicOutboundChannel")
+    },
+    incentivized: {
+        inbound: artifacts.require("IncentivizedInboundChannel"),
+        outbound: artifacts.require("IncentivizedOutboundChannel")
+    },
+}
 
-    const bridgeAbiFile = uniqueFilename(os.tmpdir(), "Bridge")
-    const ethAbiFile = uniqueFilename(os.tmpdir(), "ETHApp")
-    const erc20AbiFile = uniqueFilename(os.tmpdir(), "ERC20App")
+const channels = {
+    basic: {
+        inbound: null,
+        outbound: null
+    },
+    incentivized: {
+        inbound: null,
+        outbound: null
+    },
+}
 
-    fs.writeFileSync(bridgeAbiFile, JSON.stringify(bridge.abi, null, 2))
-    fs.writeFileSync(ethAbiFile, JSON.stringify(ethApp.abi, null, 2))
-    fs.writeFileSync(erc20AbiFile, JSON.stringify(erc20App.abi, null, 2))
+const dump = (channels) => {
 
     const config = {
         ethereum: {
             endpoint: "ws://localhost:9545/",
             "descendants-until-final": 35,
-            bridge: {
-                address: bridge.address,
-                abi: bridgeAbiFile,
-            },
-            apps:{
-                eth:{
-                    address: ethApp.address,
-                    abi: ethAbiFile,
+            channels: {
```

Comment:
need to dump apps too so that apps on substrate can verify the sending/source contract address

Reply:
ah this script is kinda unmaintained. I should delete it. The E2E tests now use dumpTestConfig.js.